airflow/providers/google/cloud/triggers/vertex_ai.py

```diff
@@ -84,16 +97,66 @@ async def run(self) -> AsyncIterator[TriggerEvent]:
             return
 
         status = "success" if job.state in self.statuses_success else "error"
-        message = f"Hyperparameter tuning job {job.name} completed with status {job.state.name}"
+        message = f"{self.job_type_verbose_name} {job.name} completed with status {job.state.name}"
         yield TriggerEvent(
             {
                 "status": status,
                 "message": message,
-                "job": HyperparameterTuningJob.to_dict(job),
+                "job": self._serialize_job(job),
             }
         )
 
-    def _get_async_hook(self) -> HyperparameterTuningJobAsyncHook:
+    async def _wait_job(self) -> Any:
+        """Awaits a Vertex AI job instance for a status examination."""
+        raise NotImplementedError
+
+    def _serialize_job(self, job: Any) -> Any:
+        return self.job_serializer_class.to_dict(job)
+
+
+class CreateHyperparameterTuningJobTrigger(BaseVertexAIJobTrigger):
+    """CreateHyperparameterTuningJobTrigger run on the trigger worker to perform create operation."""
+
+    trigger_class_path = (
+        "airflow.providers.google.cloud.triggers.vertex_ai.CreateHyperparameterTuningJobTrigger"
```

Comment:
This can be shortened by moving `airflow.providers.google.cloud.triggers` into the base class.

Just a suggestion.

Reply:
Indeed. I've done it and described in the [other message](https://github.com/VladaZakharova/airflow/pull/20#discussion_r1506077193). Thank you!